src/resolvers/noderesolver.ts

```diff
@@ -6,7 +6,7 @@ const debug = Debug("resolverengine:noderesolver");
 export function NodeResolver(): SubResolver {
   return async (what: string): Promise<string | null> => {
     try {
-      return require.resolve(what);
+      return require.resolve(what, {paths: [process.cwd()]});
```

Comment:
The whole Resolver-Engine has an option for a working directory in case you want to use a different location. The solution which works with the current API would be to use the `workingDir` from [here](https://github.com/resolver-engine/resolver-engine/blob/c79977746178614b662fae6c8d52133f23c8a83b/src/resolverengine.ts#L23)